charts/astronomer/templates/ingress.yaml

```diff
@@ -13,13 +13,18 @@ metadata:
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     heritage: {{ .Release.Service }}
   annotations:
+    {{- if .Values.global.sidecar.enabled  }}
+    kubernetes.io/ingress.class: "default"
+    route.openshift.io/termination: "reencrypt"
+    {{- else }}
```

Comment:
Could potentially decouple the openshift specifics here out to a separate flag.

Also could decouple and pass these annotations through as `extraAnnotations` on each component and document / provide sample configs.

Reply:
this change has been moved to extraAnnotations flag